src/engine/Sprite.cxx

```
@@ -46,7 +57,18 @@ void Sprite::render() const
 
       if (highlightSprite)
       {
-        SDL_SetTextureColorMod(m_SpriteData[currentLayer].texture, 255, 255, 255);
+        if (SDL_SetTextureColorMod(m_SpriteData[currentLayer].texture, 255, 255, 255))
+        {
+          // indicates failure.
```

Put at least TODO, if the error is not handled